### ISSUE TITLE
[REQUEST] 1.16+

### ISSUE THREAD

**Issue by AlexTheDolphin0** [yeelp/Distinct-Damage-Descriptions#79]
Many Minecraft mods have been updating from 1.12.2 to 1.16.5. a mod like this is a really interesting concept. I know there are a lot of code changes but could this mod possibly be updated past it's current version?

**Comment by MagmaSlime123** [comment]
Think this mod's only being worked on for 1.12, might be wrong though.

**Comment by yeelp** [comment]
So this issue is actually pretty old but my stance has never changed. Here's what I've said on my Discord about it in regards to : "is there any plan to update DDD to [a] higher version?":

> I would like to, but DDD is decently complex and it would take time to port. Last thing I'd want to do is have active development and maintenance on two versions. I'd like to have the 1.12.2 version in a stable spot where "things just work" so I don't have to flip flop between the two versions all the time. Because given the love DDD has on 1.12.2, porting to a new version would not mean abandoning the old one, contrary to what I may or may not have said in the past. I'm aware that not all mods have to be updated to newer versions and that's fine. I don't have a problem with 1.12.2. But I do know a lot of people also play newer versions of modded minecraft so getting DDD out there would be cool. I will not project when such a port will be done to avoid the "soon™️", since I really have no idea when I'd like to start it.

If/when I do port, it will likely remain as a Forge/NeoForge mod. Maintaining a Fabric version sounds like a hassle and the two versions would likely not behave the exact same way. Plus that would be yet another version to manage.

tl;dr, I'd love to at some point port DDD to a newer version, likely 1.20.1; that seems to be the most popular modern modded version from my research. But only when DDD (and my life lol) is stable. DDD actually looks pretty stable so far on its current version.

I'll be locking this and pinning this for those curious. This is where things stand right now.
